packages-content-model/roosterjs-content-model-core/lib/corePlugin/ContentModelCopyPastePlugin.ts

```diff
@@ -250,6 +256,41 @@ class ContentModelCopyPastePlugin implements PluginWithState<CopyPastePluginStat
     };
 }
 
+/**
+ * @internal
+ * Exported only for unit testing
+ */
+export function adjustSelectionForCopyCut(pasteModel: ContentModelDocument) {
+    let selectionMarker: ContentModelSelectionMarker | undefined;
+    let firstBlock: ContentModelBlock | undefined;
+
+    iterateSelections(pasteModel, (_, tableContext, block, segments) => {
+        if (selectionMarker) {
+            if (
+                tableContext &&
+                block &&
```

Comment:
Maybe we don't event need to check block here. Even block doesn't exist, we can still do the same handling.

Reply:
Agreed, removed that